content/en/platform/corda/5.0/developing-applications/tooling/_index.md

```diff
@@ -26,6 +26,6 @@ CorDapp Developers require the following:
 | Tool                                            | Description                                                                                                                                                                                                                                                                                                |
 | ----------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | CorDapp Standard Development Environment (CSDE) | The [CSDE]({{< relref "../getting-started/_index.md" >}}) guides Developers who are new to Corda 5 from setting up their development environment through to writing, compiling, and running their first basic CorDapp.                                                                                     |
-| Corda CLI                                       | The CSDE uses the Corda CLI in the background. CorDapp Developers also require the Corda CLI to [build CPB files]({{< relref "../packaging/cpb.md" >}}). For information about installation, see [Installing the Corda CLI]({{< relref "./installing-corda-cli.md" >}}).                                 |
+| Corda CLI                                       | The CSDE uses the Corda CLI in the background. CorDapp Developers also require the Corda CLI to build {{< tooltip >}}CPB{{< /tooltip >}} files. For information about installation, see [Installing the Corda CLI]({{< relref "./installing-corda-cli.md" >}}).                                 |
```

Comment:
add tooltip for CSDE

Reply:
The row immediately above in the table explains the CSDE doesn't seem necessary.